src/envVariables.js

```diff
@@ -55,31 +57,49 @@ let enviromentVariables = {
 	},
 	KAFKA_GROUP_ID: {
 		message: 'Required kafka group id',
-		optional: false,
+		optional: true,
+		default: 'mentoring',
 	},
 	NOTIFICATION_KAFKA_TOPIC: {
 		message: 'Required kafka topic',
-		optional: false,
+		optional: true,
+		default: 'notificationtopic',
+	},
+	KAFKA_TOPIC: {
+		message: 'Required kafka topic',
+		optional: true,
+		default: 'mentoring.topic',
+	},
+	RATING_KAFKA_TOPIC: {
+		message: 'Required kafka topic',
+		optional: true,
+		default: 'mentoring.rating',
+	},
+	KAFKA_MENTORING_TOPIC: {
+		message: 'Required kafka topic',
+		optional: true,
+		default: 'mentoringtopics',
 	},
 	USER_SERVICE_HOST: {
 		message: 'Required user service host',
 		optional: false,
 	},
 	USER_SERVICE_BASE_URL: {
 		message: 'Required user service base url',
-		optional: false,
+		optional: true,
+		default: '/user/',
 	},
 	BIG_BLUE_BUTTON_URL: {
 		message: 'Required big blue button url',
-		optional: false,
+		optional: process.env.DEFAULT_MEETING_SERVICE === 'OFF' ? true : false,
 	},
 	MEETING_END_CALLBACK_EVENTS: {
 		message: 'Required meeting end callback events',
 		optional: false,
 	},
 	BIG_BLUE_BUTTON_SECRET_KEY: {
 		message: 'Required big blue button secret key',
-		optional: false,
+		optional: process.env.DEFAULT_MEETING_SERVICE === 'OFF' ? true : false,
```

Comment:
Can we update the BBB related keys optional check to
process.env.DEFAULT_MEETING_SERVICE === 'BBB' ? false : true